### ISSUE TITLE
Tracker for migrating  `lock contention` flags to config in k8s upstream

### ISSUE THREAD

**Issue by ipochi** [kinvolk/lokomotive#1514]
This issue mostly acts as a tracker for driving migrating the `lock contention` kubelet flags to a config file as we are using in Lokomotive.

Part of the work is done by an external person. This work is now upon us to get the feature merged.

1/3 PRs are merged, the effort centers around merging the remaining 2 existing PRs or closing them and opening new ones.

**Comment by ipochi** [comment]
Update:

* I created a PR https://github.com/kubernetes/test-infra/pull/22758  to supersede https://github.com/kubernetes/test-infra/pull/20106. I reached out to the author of the original PR and he was kind enough to close his PR in favor of mine. 

https://github.com/kubernetes/test-infra/pull/22758 was merged few days ago.

* I created another PR with the actual test in https://github.com/kubernetes/kubernetes/pull/103608 which was created to supersede https://github.com/kubernetes/kubernetes/pull/96775

The new PR takes a different approach thereby making already merged PR (https://github.com/kubernetes/kubernetes/pull/97028) not needed for this feature.

I have provided more details  on https://github.com/kubernetes/kubernetes/pull/103608 and created a follow up PR https://github.com/kubernetes/test-infra/pull/22758. If agreed that https://github.com/kubernetes/kubernetes/pull/97028 is not needed then I will create another PR reverting the said change. 

/cc @rata @iaguis

**Comment by iaguis** [comment]
We're pending on reviews on a couple of PRs upstream.
